examples/nodefeaturegroup.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: nfd.k8s-sigs.io/v1alpha1
+kind: NodeFeatureGroup
+metadata:
+  name: node-feature-group-1
+spec:
+  featureGroupRules:
+    - name: "e2e-attribute-test-1"
```

nit: sync with what we have in docs

```suggestion
metadata:
  name: node-feature-group-example
spec:
  featureGroupRules:
    - name: "kernel version"
```